pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.ecocode</groupId>
+    <artifactId>ios-plugin</artifactId>
+    <version>1.0.2-SNAPSHOT</version>
```

Comment:
change version to `1.1.0-SNAPSHOT` beacause of important migration but with retro-compatibility